base-notebook/Dockerfile

```diff
@@ -28,13 +28,14 @@ ENV DEBIAN_FRONTEND noninteractive
 RUN apt-get update --yes && \
     apt-get upgrade --yes && \
     apt-get install --yes --no-install-recommends \
-    tini \
-    wget \
     ca-certificates \
-    sudo \
-    locales \
     fonts-liberation \
-    run-one && \
+    locales \
+    pandoc \
```

Comment:
Sorted, and added this line.
Pandoc is not installed by default in arm images.

Reply:
Can you make inline note about that?

This is knowledge that can be useful to retain like an inline comment i think

Reply:
Done, please, take a look